src/app/app.module.ts

```diff
@@ -58,7 +61,10 @@ import {AutosizeDirective} from './components/textarea-autosize/textarea-autosiz
     FormsModule,
     ReactiveFormsModule,
     AppTranslateModule,
-    AppRoutingModule
+    AppRoutingModule,
+    AgmCoreModule.forRoot({
+      apiKey: 'AIzaSyABLDG-8LXDMSp6vsTNNI2uSPCWA4eNs-Y'
```

Comment:
I tried but since the method forRoot has to be called when the module is registered at the root level i don't know how to solve this other than extracting the key.